### PR TITLE
- fixing #389: one way handling for bike profiles

### PR DESCRIPTION
**Pull request by marq24**
one way handling for bike profiles

please delete `389-fix-bike_oneway_handling` branch after merge

### Pull Request Checklist
- [X] 1. I have rebased the latest version of the master branch into my feature branch and all conflicts have been resolved.
- [X] 2. I have added information about the change/addition to functionality to the CHANGELOG.md file under the [Unreleased] heading.
- [X] 4. I have removed unnecessary commented out code, imports and System.out.println statements.
- [X] 8. I have built graphs with my code of the Heidelberg.osm.gz file and run the api-tests with all test passing
- [X] 9. I have referenced the Issue Number in the Pull Request (if the changes were from an issue).
- [X] 12. I have written in the Pull Request information about the changes made including their intended usage and why the change was needed.

Fixes # .
#389 

### Examples and reasons for differences between live ORS routes and those generated from this pull request
- as described in #389 

### Required changes to app.config (if applicable)
- NONE

### CODE REVIEW

File: openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java [hunk 1]
```diff
@@ -700,13 +700,13 @@ void collect(ReaderWay way, double wayTypeSpeed, TreeMap<Double, Integer> weight
         }
 
         // MARQ24 MOD START
```
Comment (aoles) [inline openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java]: Is the `// MARQ24 MOD` comment still valid after the update?

**Review by aoles — CHANGES_REQUESTED** [review]
Thanks a lot @marq24 for your contribution! Please consider my inline comment.

Did you have a chance to build and test with the [relevant OSM dataset](http://download.geofabrik.de/europe/netherlands/limburg.html) that this in fact fixes this particular routing problem? Unless you have performed some other more general testing, it would be good to verify the correctness using at least the example provided in the issue.

Cheers!

**Comment by marq24** [comment]
& sure it have been tested on the specific route...
image

File: openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java (hunk 1 above)
Comment (marq24) [reply openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/flagencoders/bike/CommonBikeFlagEncoder.java]: sorry - done

**Review by aoles — APPROVED** [review]

**Comment by aoles** [comment]
Thanks again @marq24! It's always a pleasure to work with you :)

**Pull request merged**